app/routes/resource/get-recent-front-matter.ts

```diff
@@ -0,0 +1,22 @@
+import { type LoaderArgs } from "@remix-run/server-runtime";
+import type {
+  PostFrontMatterCollection,
+  PostFrontMatterWithSlug,
+} from "blog/models";
+import { FRONT_MATTER_CACHE_FILENAME } from "blog/paths";
+
+export const loader = async function ({
+  request,
+}: LoaderArgs): Promise<PostFrontMatterWithSlug> {
+  const url = new URL(FRONT_MATTER_CACHE_FILENAME, new URL(request.url).origin);
+  return fetch(url)
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error(`Could not retrieve posts. ${response.statusText}`);
+      }
+      return response.json() as Promise<PostFrontMatterCollection>;
+    })
+    .then((frontMatterCollection) => {
+      return frontMatterCollection[0];
```

Comment:
so this returns only the single most recent? 

actually, it doesn't seem like this endpoint is used. this needed?

Reply:
It's not used, but we may want to keep it. If we get more blog posts, it may be helpful to have a route that plucks the most recent one so we can put that info on the homepage or something. Seem like a good idea?